### DISCUSSION TITLE
How Do I Render an HTML String as HTML in a Component?

### DISCUSSION THREAD

**Discussion by zicklag** [preactjs/preact#3993] [Q&A] [answered]
Hey there! I'm looking for a simple equivalent to `v-html` in Preact.

I have a string with HTM in it, I just want it to spit that HTML out as HTML. ( I know it's "dangerous", but not in my scenario when I'm getting the HTML from a sanitizing markdown parser for markdown that I, the developer, actually wrote in the first place. )

**Comment by marvinhagemeister** [comment, marked as answer]
You can inject HTML strings via the `dangerouslySetInnerHTML` prop:

```jsx
function RenderHTML() {
  const markup = { __html: '<p>some raw html</p>' };
  return <div dangerouslySetInnerHTML={markup} />;
}
```

**Reply by sereitepy** [reply]
thank you thank you
